distributed/deploy/ssh.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         self.kwargs = copy.copy(kwargs)
         self.name = name
         self.remote_python = remote_python
-        self.nprocs = self.kwargs.pop("nprocs", 1)
+        self.num_workers = self.kwargs.pop("num_workers", 1)
```

Comment:
@jrbourbeau I just realized that this is a breaking change if anyone is using the `Worker` class directly. I'll add similar deprecation checks here as elsewhere.